gcp/appengine/frontend3/src/styles.scss

```diff
@@ -669,6 +669,10 @@ dl.vulnerability-details,
   }
 }
 
+.vulnerability-package-subsection dl:not(:last-child) {
+  margin-bottom: 15px;
```

Comment:
Can you put this in the nested area you had it in before, and use the & operator to apply the last-child property.

See https://sass-lang.com/documentation/style-rules/parent-selector/

Reply:
Thanks!